tests/test_sockets.py

```diff
@@ -1526,6 +1530,65 @@ async def test_send_after_close(self, family: AnyIPAddressFamily) -> None:
             await udp.send(b"foo")
 
 
+@pytest.mark.network
+@pytest.mark.usefixtures("check_asyncio_bug")
```

Comment:
apparently this is fixed in https://github.com/python/cpython/issues/83329

Reply:
done

Reply:
ah it's not actually fixed, I tried it out

Reply:
but the tests pass, so should I add it again or not?

Reply:
That doesn't explain why _all_ the tests pass.